UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/InvalidBundleInfoScenariosForLargeMediaExchangeSet.cs

```diff
@@ -0,0 +1,54 @@
+﻿using NUnit.Framework;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.FunctionalTests.Helper;
+using System.Net.Http;
+using System.Collections.Generic;
+using UKHO.ExchangeSetService.API.FunctionalTests.Models;
+
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.FunctionalTests
+{
+    public class InvalidBundleInfoScenariosForLargeMediaExchangeSet
+    {
+        private string EssJwtToken { get; set; }
+        private string FssJwtToken { get; set; }
+        private ExchangeSetApiClient ExchangeSetApiClient { get; set; }
+        private FssApiClient FssApiClient { get; set; }
```

Comment:
Is the FSSApiClient used?

If not can it be removed?

Reply:
Hi Martyn,

The unused code has been removed.

Regards,
Mayuresh Gawde